src/router/index.ts

```diff
@@ -43,6 +43,27 @@ const router = createRouter({
       path: "/user",
       name: "UserPage",
       component: () => import("@/views/UserPage.vue"),
+      meta: {
+        requiresAuth: true,
+      },
+    },
+    {
+      path: "/empl",
```

Comment:
Why not something prettier 😄 
```suggestion
      path: "/employee",
```

Reply:
prettier is always better 😋